pkg/kubecost/totals _json.go

```diff
@@ -0,0 +1,77 @@
+package kubecost
```

Comment:
This file name looks really weird. There appears to be a character in between "s" and "_".

Reply:
!! woah

Reply:
Will fix ty

Reply:
Fixed!